mautic/mautic
# Normalize report data If contact or company use bool/select/multisele…

…ct options

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR added support to normalize contacts/company boolean/select/multiselect fields in reports. It displays the value of these types of fields. After PR it should display labels to be more ready friendly



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create report for contacts and choose boolean/select/multiselect field in columns
3. Check If after PR it displays labels of options (for booleans instead of 1 display Yes....)
